### PR TITLE
Minor layout changes with addition of direct download links to release artifacts

### PR DESCRIPTION
**Pull request by bbarani**
*Issue #, if available:*

*Description of changes:*

   This commit adds
     * Getting started guide link to download overview content
     * Direct download link for tar, exe, zip files (Open distro for Elasticsearch and Kibana artifacts)
     * New icons for windows, zip and ODBC drivers
     * links to SHA and install guides corresponding to the artifacts
     * Additional info section in download page for release notes, version history and plugin installation guide

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: _data/pages/es/downloads.yml [hunk 1]
```diff
@@ -12,6 +17,9 @@ images:
   tar: "/assets/media/icons/tar.png"
   jdbc: "/assets/media/icons/javajdbc.png"
   perftop: "/assets/media/icons/consol.png"
+  windows: "/assets/media/icons/windows.png"
+  zip: "/assets/media/icons/zip.png"
+  odbc: "/assets/media/icons/odbc.png"
```
Comment (aetter) [inline _data/pages/es/downloads.yml:22]: I'd resize this image in the source or in the CSS, but it's really large on screen. ~60-70% of its current size seems appropriate.

File: _data/pages/es/downloads.yml [hunk 2]
```diff
@@ -1,9 +1,14 @@
 banner:
-  title: Download
+  title: Downloads
 
 intro:
-  title: Get started
-  ext_content: We provide Docker images, RPM files, Debian packages, and more.
+  title: Ready to get started?
+  ext_content: We provide Docker images, RPM files, Debian packages and more. Jump to <a href="https://opendistro.github.io/for-elasticsearch-docs/">getting started guide</a> to dive deep in to Open distro for Elasticsearch.
```
Comment (aetter) [inline _data/pages/es/downloads.yml:6]: I think we can omit the second sentence here, but if you want to include it, I recommend something like:

`See the <a href="https://opendistro.github.io/for-elasticsearch-docs/">documentation</a> for a deep dive into Open Distro for Elasticsearch.`

File: _layouts/downloads.html [hunk 3]
```diff
@@ -30,24 +30,30 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.elasticsearch.title}}</h1>
...
         <div class="row justify-content-center text-center px-2">
-          <div class="col-12 col-md-10"> </div>
+          <div class="col-12 col-md-10"></div>
           <div class="col-6 col-md-4 pt-3 pb-5 link-on-white-paragraph">
-            <a href="{{site.data.pages.es.downloads.elasticsearch.rpm_url}}" class="software-download" target="about:blank" onclick="trackOutboundLink('{{site.data.pages.es.downloads.elasticsearch.rpm_url}}');">
-              <h7 style="font-weight:bold;">RPM install guide</h7>
+            <a href="{{site.data.pages.es.downloads.elasticsearch.rpm_url}}" class="software-download"
+               target="about:blank"
+               onclick="trackOutboundLink('{{site.data.pages.es.downloads.elasticsearch.rpm_url}}');">
+              <h7 style="font-weight:bold;">RPM Install guide</h7>
```
Comment (aetter) [inline _layouts/downloads.html:49]: "Install guide" is capitalized differently throughout this page. We use sentence case whenever possible, so RPM install guide, Debian install guide, Install guide, etc.

File: _layouts/downloads.html [hunk 4]
```diff
@@ -56,32 +62,99 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.elasticsearch.title}}</h1>
...
+                 onclick="trackOutboundLink('{{site.data.pages.es.downloads.elasticsearch.tar_checksum_url}}-{{site.data.pages.es.downloads.version.odfe}}.tar.gz.sha512');">
+                <h7 style="color:#2070C4;font-size:12px;">SHA</h7>
+              </a>
+              <h7 class="vertical-line"></h7>
+              <a href="{{site.data.pages.es.downloads.elasticsearch.tar_guide_url}}"
+                 class="software-download"
+                 target="about:blank"
+                 onclick="trackOutboundLink('{{site.data.pages.es.downloads.elasticsearch.tar_guide_url}}');">
+
+                <h7 style="color:#2070C4;font-size:12px;">Install Guide</h7>
```
Comment (aetter) [inline _layouts/downloads.html:106]: Install guide

File: _layouts/downloads.html [hunk 5]
```diff
@@ -90,85 +163,137 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.elasticsearch.title}}</h1>
...
-            <a href="{{site.data.pages.es.downloads.kibana.rpm_url}}" class="software-download" target="about:blank" onclick="trackOutboundLink('{{site.data.pages.es.downloads.kibana.rpm_url}}');">
-
-
-              <h7 style="font-weight:bold;">RPM install guide</h7>
-
-
+            <a href="{{site.data.pages.es.downloads.kibana.rpm_url}}" class="software-download"
+               target="about:blank"
+               onclick="trackOutboundLink('{{site.data.pages.es.downloads.kibana.rpm_url}}');">
+              <h7 style="font-weight:bold;">RPM Install guide</h7>
```
Comment (aetter) [inline _layouts/downloads.html:190]: RPM install guide

File: _layouts/downloads.html [hunk 6]
```diff
@@ -90,85 +163,137 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.elasticsearch.title}}</h1>
...
+                 target="about:blank"
+                 onclick="trackOutboundLink('{{site.data.pages.es.downloads.kibana.tar_checksum_url}}-{{site.data.pages.es.downloads.version.odfe}}.tar.gz.sha512');">
+                <h7 style="color:#2070C4;font-size:12px;">SHA</h7>
+              </a>
+              <h7 class="vertical-line"></h7>
+              <a href="{{site.data.pages.es.downloads.kibana.tar_guide_url}}" class="software-download"
+                 target="about:blank"
+                 onclick="trackOutboundLink('{{site.data.pages.es.downloads.kibana.tar_guide_url}}');">
+
+                <h7 style="color:#2070C4;font-size:12px;">Install Guide</h7>
```
Comment (aetter) [inline _layouts/downloads.html:245]: Install guide

File: _data/pages/es/downloads.yml [hunk 7]
```diff
@@ -12,6 +17,9 @@ images:
   tar: "/assets/media/icons/tar.png"
   jdbc: "/assets/media/icons/javajdbc.png"
   perftop: "/assets/media/icons/consol.png"
+  windows: "/assets/media/icons/windows.png"
+  zip: "/assets/media/icons/zip.png"
```
Comment (aetter) [inline _data/pages/es/downloads.yml:21]: It's personal preference, of course, but I think the green icon is super garish. Maybe @stevensideyliu has an opinion.

File: _layouts/downloads.html [hunk 8]
```diff
@@ -294,13 +421,45 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.perftop.title}}</h1>
...
+          What's in this release? View the
+          <a href="{{site.data.pages.es.downloads.misc.release_notes}}-{{site.data.pages.es.downloads.version.odfe}}.md"
+             target="about:blank"
+             onclick="trackOutboundLink('{{site.data.pages.es.downloads.misc.release_notes}}-{{site.data.pages.es.downloads.version.odfe}}.md');">
+            <h7 style="font-weight:bold;">release notes</h7>
+          </a>
+        </div>
+        <div class="link-on-white-paragraph">
+          Are you looking for a different version? View
+          <a href="{{site.data.pages.es.downloads.misc.previous_releases}}"
```
Comment (aetter) [inline _layouts/downloads.html:444]: `Looking for an older version? View <a href="{{site.data.pages.es.downloads.misc.previous_releases}}.`

File: _layouts/downloads.html [hunk 9]
```diff
@@ -294,13 +421,45 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.perftop.title}}</h1>
...
+      </div>
+    </div>
+
+
+    <!-- Misc section -->
+    <div class="row px-3 justify-content-center text-center px-2">
+      <div class="col-12 col-md-12"
+           style="background-color:white; border-style: solid; border-color:#b4bcc2; border-width: 1px;"></br>
+        <div class="link-on-white-paragraph">
+          What's in this release? View the
```
Comment (aetter) [inline _layouts/downloads.html:435]: `What's new in this release? See the`

File: _layouts/downloads.html [hunk 10]
```diff
@@ -294,13 +421,45 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.perftop.title}}</h1>
...
+          <a href="{{site.data.pages.es.downloads.misc.release_notes}}-{{site.data.pages.es.downloads.version.odfe}}.md"
+             target="about:blank"
+             onclick="trackOutboundLink('{{site.data.pages.es.downloads.misc.release_notes}}-{{site.data.pages.es.downloads.version.odfe}}.md');">
+            <h7 style="font-weight:bold;">release notes</h7>
```
Comment (aetter) [inline _layouts/downloads.html:439]: Period at the end.

File: _layouts/downloads.html [hunk 11]
```diff
@@ -294,13 +421,45 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.perftop.title}}</h1>
...
+          </a>
+        </div>
+        <div class="link-on-white-paragraph">
+          Are you looking for a different version? View
+          <a href="{{site.data.pages.es.downloads.misc.previous_releases}}"
+             target="about:blank"
+             onclick="trackOutboundLink('{{site.data.pages.es.downloads.misc.previous_releases}}');">
+            <h7 style="font-weight:bold;"> previous releases</h7>
+          </a></div>
+        <div class="link-on-white-paragraph">Interested in standalone plugins? Visit Standalone Elasticsearch
```
Comment (aetter) [inline _layouts/downloads.html:449]: `Prefer the standalone plugins? See the install guide.`

**Review by aetter — CHANGES_REQUESTED** [review]

File: _data/pages/es/downloads.yml (hunk 2 above)
Comment (bbarani) [reply _data/pages/es/downloads.yml]: I have made the suggested changes.

File: _layouts/downloads.html (hunk 3 above)
Comment (bbarani) [reply _layouts/downloads.html]: I have made the suggested changes.

File: _layouts/downloads.html (hunk 5 above)
Comment (bbarani) [reply _layouts/downloads.html]: I have made the suggested changes.

File: _layouts/downloads.html (hunk 4 above)
Comment (bbarani) [reply _layouts/downloads.html]: I have made the suggested changes.

File: _data/pages/es/downloads.yml (hunk 7 above)
Comment (bbarani) [reply _data/pages/es/downloads.yml:21]: I am good with using some other icon as well. Let me know!

File: _layouts/downloads.html (hunk 8 above)
Comment (bbarani) [reply _layouts/downloads.html:443]: I have made the suggested changes.

File: _data/pages/es/downloads.yml (hunk 1 above)
Comment (bbarani) [reply _data/pages/es/downloads.yml:22]: I have resize the image now.

File: _layouts/downloads.html (hunk 9 above)
Comment (bbarani) [reply _layouts/downloads.html]: Updated it as suggested.

File: _layouts/downloads.html (hunk 10 above)
Comment (bbarani) [reply _layouts/downloads.html]: Updated it as suggested.

File: _layouts/downloads.html (hunk 11 above)
Comment (bbarani) [reply _layouts/downloads.html]: Updated it as suggested.

File: _layouts/downloads.html (hunk 6 above)
Comment (bbarani) [reply _layouts/downloads.html]: Updated it as suggested.

File: _data/pages/es/downloads.yml (hunk 7 above)
Comment (bbarani) [reply _data/pages/es/downloads.yml:21]: @stevensideyliu, can you provide your feedback?

Comment (bbarani) [reply _data/pages/es/downloads.yml:21]: Updated the icon for zip to use this (icon)[http://www.softicons.com/system-icons/nano-icon-set-by-freeman/zip-files-2-icon]

**Review by aetter — APPROVED** [review]

**Review by stevensideyliu — COMMENTED** [review]
This looks good.

**Pull request merged**